.editorconfig

```diff
@@ -0,0 +1,12 @@
+[*]
+insert_final_newline = true
+
+[*.rs]
+indent_style = space
+indent_size  = 4
+trim_trailing_whitespace = true
+
+[*.yml]
```

Comment:
I believe our config files are expected to be `.conf`; should we be making them `.yml` instead?

Reply:
Probably, but that line is mostly for correct indentation of the Github workflow files.